openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1053,7 +1053,7 @@ Below is a non-normative example of a `proofs` parameter:
 
 A key attestation in JWT format as defined in (#keyattestation-jwt).
 
-When a key attestation is used as a proof type, it MUST contain the `c_nonce` value provided by the Credential Issuer in its `nonce` parameter.
+If the Credential Issuer has a Nonce Endpoint (as defined in (#nonce-endpoint)), the `c_nonce` value provided by the Credential Issuer MUST be provided in the key attestation's `nonce` parameter.
```

Comment:
This is a normative change. Should we mandate nonce endpoint in this case?

Reply:
I agree its better to be consistent here unless there is compelling reason not to be.

Reply:
We don't have any freshness guarantee with this change. Yet again, it's also optional for jwt proof type. So I think I'm okay with this. 

cc @c2bo